### ISSUE TITLE
Error registering pipeline secret when uppercase

### ISSUE THREAD

**Issue by chobdayIbm** [ibm-garage-cloud/ibm-garage-cloud-cli#1]
I followed the guide in https://ibm-garage-cloud.github.io/ibm-garage-developer-guide/getting-started/deploy-app,

When trying to run `igc pipeline`
I got 
```
Error registering pipeline: Secret "chobdayIbm.angular-quickstart" is invalid: metadata.name: Invalid value: "chobdayIbm.angular-quickstart": a DNS-1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')
```

I suppose this is trying to run `kubectl create secret`, and the problem is because my github id has an upper case character.  
I got this even when I used username chobday.   
Suggestion : 
1. Have a prompt for the name of the secret
2. Sanitize the name, e.g. convert to lower case

**Comment by seansund** [comment]
I believe this was addressed in v0.2.4 of the cli. I'll confirm

**Comment by seansund** [comment]
Nope. still doesn't work... I'll fix it
